test_util/convertECL.cpp

```diff
@@ -356,30 +367,34 @@ int main(int argc, char **argv) {
     }
 
     if (formattedOutput) {
-
         auto search = to_formatted.find(extension);
-
         if (search != to_formatted.end()){
             resFile = rootN + search->second;
-        } else if (extension.substr(1,1)=="X"){
+        }
+        else if (extension.substr(1,1) == "X") {
             resFile = rootN + ".F" + extension.substr(2);
-        } else if (extension.substr(1,1)=="S"){
+        }
+        else if (extension.substr(1,1) == "S") {
             resFile = rootN + ".A" + extension.substr(2);
-        } else {
+        }
+        else {
             std::cout << "\n!ERROR, unknown file type for input file '" << rootN + extension << "'\n" << std::endl;
             exit(1);
         }
-    } else {
-
+    }
+    else {
         auto search = to_binary.find(extension);
 
         if (search != to_binary.end()){
             resFile = rootN + search->second;
-        } else if (extension.substr(1,1)=="F"){
+        }
+        else if (extension.substr(1,1)=="F"){
```

Comment:
spacing while at it (and below)

Reply:
> spacing while at it (and below)

Right.  I've pushed an update to that effect, including in a few other locations in the file.